embassy-stm32/src/rcc/mod.rs

```diff
@@ -97,8 +97,11 @@ pub mod low_level {
 }
 
 pub(crate) mod sealed {
+    use super::Clocks;
+
     pub trait RccPeripheral {
         fn frequency() -> crate::time::Hertz;
+        fn clocks() -> &'static Clocks;
```

Comment:
Why is this method needed? It does not make much sense to have it in `RccPeripheral`. "Get *my* clock" makes sense (that's what `frequency()` does) but "get *all* clocks" doesn't.